{"name": "Verify login functionality", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'sauce_username.txt'", "trace": "self = <test_saucedemo_ui.TestSaucedemo.TestUserAuthentication object at 0x108750130>, start_sauce_instance = <modules.ui.page_objects.saucedemo.StartSaucedemo object at 0x108750dc0>\n\n    @allure.title(\"Verify login functionality\")\n    @pytest.mark.sauce\n    def test_login(self, start_sauce_instance):\n>       with open(\"sauce_username.txt\", \"r\") as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'sauce_username.txt'\n\ntests/ui/test_saucedemo_ui.py:15: FileNotFoundError"}, "start": 1707770529042, "stop": 1707770529042, "uuid": "0821cc1a-7682-4791-a60c-40555346bfc2", "historyId": "65e85a0bccfe9ecdd1292c20722f1f9f", "testCaseId": "65e85a0bccfe9ecdd1292c20722f1f9f", "fullName": "tests.ui.test_saucedemo_ui.TestUserAuthentication#test_login", "labels": [{"name": "suite", "value": "Saucedemo"}, {"name": "feature", "value": "User Authentication"}, {"name": "tag", "value": "sauce"}, {"name": "parentSuite", "value": "tests.ui"}, {"name": "subSuite", "value": "TestSaucedemo"}, {"name": "host", "value": "EPUAVINW0156"}, {"name": "thread", "value": "23230-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_saucedemo_ui"}]}